lib/completion.js

```diff
@@ -42,17 +42,18 @@ const isWindowsShell = require('./utils/is-windows-shell.js')
 const output = require('./utils/output.js')
 const fileExists = require('./utils/file-exists.js')
 
-const usageUtil = require('./utils/usage.js')
 const { promisify } = require('util')
+const BaseCommand = require('./base-command.js')
 
-class Completion {
-  constructor (npm) {
-    this.npm = npm
+class Completion extends BaseCommand {
+  /* istanbul ignore next - see test/lib/load-all-commands.js */
+  static get name () {
+    return 'completion'
   }
 
   /* istanbul ignore next - see test/lib/load-all-commands.js */
-  get usage () {
-    return usageUtil('completion', 'source <(npm completion)')
+  static get usage () {
+    return ['| source']
```

Comment:
This doesn't actually work on bash, though.  `source` requires a filename.

```
$ npm completion | source
bash: source: filename argument required
source: usage: source filename [arguments]
```

Could replace with:

```js
static get usage () {
  return ['| source <(cat)']
}
```

Reply:
Technically none of this is about the arguments. This probably belongs in `description`.

Reply:
changed this to a `description` that says
> npm command completion script. save to ~/.bashrc or ~/.zshrc

We should have a separate bikeshedding talk to see if the usage output could be more clear.